packages/react/src/patterns/sub-patterns/LinkList/LinkList.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { CTA } from '../../../components/CTA';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * LinkList Component, for use with content arrays
+ *
+ * @param {object} props props object
+ * @param {string} props.heading  Heading string
+ * @param {Array} props.items array of item
+ * @returns {*} JSX LinkList component
+ */
+const LinkList = ({ heading, items }) => {
+  return (
+    <div
```

Comment:
@photodow is it possible that the LinkList can be utilized as not an aside? Just so happens that in the designs it is a right-aligned component, but seems like it might be flexible enough to be a component that is at the bottom of content, but not sure is that is the design intent or not?

Reply:
I don't know. That's more of a question for @wonilsuhibm and @larahanlon2 I would guess. I pictured it only used in scenarios like this off to the side. But if we want to leave the door open and avoid pidgeon holing it into an `aside` I guess I can understand that reasoning a little.